### PR TITLE
Update helusers from 0.8.1 to 0.9.0

### PR DESCRIPTION
**Pull request by mikkokeskinen**
The new helusers allows example backend to accept API Tokens from both Tunnistamo and Keycloak.

Additionally, change environ settings to use lists in the following environment keys:

- OIDC_AUDIENCE
- OIDC_API_SCOPE_PREFIX
- OIDC_API_AUTHORIZATION_FIELD
- OIDC_ISSUER

Thus allowing the configuration of both Tunnistamo and Keycloak at the same time.

### CODE REVIEW

File: example_backend_profile/settings.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
     DEBUG=(bool, True),
     SECRET_KEY=(str, ""),
     ALLOWED_HOSTS=(list, []),
-    OIDC_AUDIENCE=(str, ""),
-    OIDC_API_SCOPE_PREFIX=(str, ""),
-    OIDC_API_AUTHORIZATION_FIELD=(str, ""),
+    OIDC_AUDIENCE=(list, []),
+    OIDC_API_SCOPE_PREFIX=(str, []),
```
Comment (akikoskinen) [inline example_backend_profile/settings.py:13]: `OIDC_API_SCOPE_PREFIX` type is still `str`.

File: requirements.txt [hunk 2]
```diff
@@ -39,7 +39,7 @@ django-cors-headers==3.10.0
     # via -r requirements.in
 django-environ==0.7.0
     # via -r requirements.in
-django-helusers==0.8.1
+django-helusers==0.9.0
```
Comment (akikoskinen) [inline requirements.txt:42]: Is it now possible to remove the `drf-oidc-auth` dependency?

File: example_backend_profile/settings.py (hunk 1 above)
Comment (mikkokeskinen) [reply example_backend_profile/settings.py]: Ah! Fixed.

File: requirements.txt (hunk 2 above)
Comment (mikkokeskinen) [reply requirements.txt:42]: It's required by helsinki-profile-gdpr-api.

Comment (akikoskinen) [reply requirements.txt:42]: But can be removed from `requirements.in`?

Comment (mikkokeskinen) [reply requirements.txt:42]: True that.

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**